config/configtest/configtest.go

```diff
@@ -40,3 +49,131 @@ func LoadConfigAndValidate(fileName string, factories component.Factories) (*con
 	}
 	return cfg, cfg.Validate()
 }
+
+// ValidateConfig enforces that given configuration object is following the patterns
+// used by the collector. This ensures consistency between different implementations
+// of components and extensions. It is recommended for implementers of components
+// to call this function on their tests passing the default configuration of the
+// component factory.
+func ValidateConfig(config interface{}) error {
```

Comment:
s/ValidateConfig/CheckConfigStruct?

Reply:
Updated

Reply:
@Aneurysm9 I asked for the change only of this function name not every time "ValidateConfig" is used.